app/src/main/kotlin/maru/app/MaruAppCli.kt

```diff
@@ -56,17 +56,22 @@ enum class Network(
   descriptionHeading = "%nDescription:%n%n",
   optionListHeading = "%nOptions:%n",
   footerHeading = "%n",
+  mixinStandardHelpOptions = true,
 )
 class MaruAppCli(
-  private val dryRun: Boolean = false,
+  private val maruAppFactory: MaruAppFactory = MaruAppFactory(),
```

Comment:
That's not a clean dependency injection without a separate interface for `MaruAppFactory`

The symptom of it is that`MaruAppFactory::create` returns an `LongRunningCloseable`, instead of a more specific type of `MaruApp`. This leads to `as MaruApp` type casts in `TestMaruAppFactory.kt` and `testutils.maru.MaruFactory::buildApp`, for example